backend/src/modules/favourite/controller/favourite/favourite.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller, Patch, UseGuards, Request, Body, Put, Get } from '@nestjs/common';
+import * as express from 'express';
+
+import { FavouriteService } from '../../service/favourite/favourite.service';
+import { AuthGuard } from '../../../../modules/authentication/guards/auth/auth.guard';
+import { FavouriteItem } from '../../models/favourite.interface';
+import { userIdCookieKey } from '../../../../shared/constants';
```

Comment:
It would be nice to import constants like express imported. So, we will see that it is constant at everywhere